### ISSUE TITLE
How to get better accuracy Yolov3? 

### ISSUE THREAD

**Issue by kartikmalhotra** [AlexeyAB/darknet#4749]
I am training yolo for a single class ( license plate) 
but the predictions are not far more accurate and the dataset I am using is annotated consisting of 1200 images.

yolov3.cfg paramters are
batch=64
subdivisions=8
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches =20000
policy=steps
steps=18000
scales=.1,.1

Help is appreciated!

**Comment by dexception** [comment]
Read the accuracy chart:
https://github.com/AlexeyAB/darknet/issues/3114
